languages/ruby/exercises/concept/ostruct/.docs/introduction.md

```diff
@@ -0,0 +1,33 @@
+Ruby comes with a Standard Library - a collection of classes for working with things such as dates, json, and networking.
+The also provide some useful functionality for making your code easier to work with.
+
+`OpenStruct` is part of the standard library and allows you to easily create an object from a hash. Rather than having to access using hash keys, `OpenStruct` instead allows us to use methods to access and set values.
+
+When using classes from the Standard Library, or any other library, you need to require that class using the `require` method.
```

Comment:
```suggestion
When using classes that are not from the Core Library, such as from Standard Library, we need to require that class using the `require` method.
```

Reply:
This implies any other libraries, and gives Standard Library as an example of a "Not Core Library".